styles/widgets/android5/radioButton.android5.less

```diff
@@ -29,7 +29,7 @@
     .transition(~"transform .14s");
 }
 
-.dx-radiobutton-checked {
+.dx-radiobutton-checked > .dx-radio-value-container {
```

Comment:
Maybe it would be better to add new class to the icon container or value-container? Try not to make the selector more specific.